examples/bri-3/src/bri/zkp/components/merkleTree/tree.d.ts

```diff
@@ -0,0 +1,6 @@
+import { PreciseProofs } from "ew-precise-proofs-js";
+
+export type Tree = {
```

Comment:
why using type here instead of interface?

Reply:
same thought